src/test/java/com/taskadapter/redmineapi/CustomFieldDefinitionsTest.java

```diff
@@ -44,6 +47,13 @@ public void testGetCustomFields() throws RedmineException {
                 assertEquals(cfd.getCustomizedType(), "issue");
                 assertEquals(cfd.getName(), "custom_boolean_1");
                 assertEquals(cfd.getFieldFormat(), "bool");
+            } else if (cfd.getId() == 3) {
+                assertEquals(cfd.getCustomizedType(), "issue");
+                assertEquals(cfd.getName(), "custom_multi_list");
+                assertEquals(cfd.getFieldFormat(), "list");
+                assertEquals(cfd.getDefaultValue(), "V2");
```

Comment:
this and other assert() calls are not correct. the expected format is (expected, actual) while this test provides (actual, expected).
I migrated many of these old-style assert() calls to assertThat() to avoid this issue (not all of them yet though). I will change these as well.